### PR TITLE
EVG-7936 Add query to get user settings 

### PR DESCRIPTION
**Pull request by khelif96**

### CODE REVIEW

File: graphql/tests/userSettings/data.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (khelif96) [inline graphql/tests/userSettings/data.json:1]: not really used but the test suite needs at least some data in here to run.

File: graphql/integration_atomic_test.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func TestAtomicGQLQueries(t *testing.T) {
 func setup(t *testing.T, directory string) atomicGraphQLState {
 	const apiKey = "testapikey"
 	const apiUser = "testuser"
-
+	const slackUsername = "testslackuser"
```
Comment (john-m-liu) [inline graphql/integration_atomic_test.go:55]: This should be set in the test data json file shouldn't it?

Comment (khelif96) [reply graphql/integration_atomic_test.go:55]: This test relies on information for the current user which is only created over here. Im not sure if its possible to update the current users information when loading in data from the data.json file. 

https://github.com/evergreen-ci/evergreen/blob/61b608ad5036afcfd40eae6c919be16f44e8c21b/graphql/integration_atomic_test.go#L135 
The test suite looks like it only performs an `insertMany` into the db based off the information in the `data.json`. I dont think its possible to update the current users information based off this setup, not sure tho. What do you think?

Comment (john-m-liu) [reply graphql/integration_atomic_test.go:55]: Oh yea Arjun ran into this before and I made the same comment. In real evergreen the user is retrieved from the db which is what I assumed is what happens here, however the test actually sets up a mock authentication system that does not maintain users in the db

**Pull request merged**